### DISCUSSION TITLE
Limiting case 1.

### DISCUSSION THREAD

**Discussion by JordanEmil** [Part-II-Computational-Physics/.github#69] [General]
I can't seem to get this working....

<img width="884" alt="Screenshot 2023-03-09 at 6 58 29 pm" src="https://user-images.githubusercontent.com/91506272/224127425-ece02aa4-643f-4d3d-8fc7-2dd5276560b8.png">

**Comment by DLHvdH** [comment]
Your `avg_times` seems to have only a few values, some of which are so low that the system likely hasn't equilibrated yet. This might affect your results. The first exercise should have given you an idea of how long it takes for the system to equilibrate.
